source/gui/_addonStoreGui/viewModels/store.py

```diff
@@ -313,22 +347,122 @@ def _handleEnableDisable(self, listItemVM: AddonListItemVM[_AddonManifestModel],
 		listItemVM.status = getStatus(listItemVM.model)
 		self.refresh()
 
-	def enableOverrideIncompatibilityForAddon(self, listItemVM: AddonListItemVM[_AddonManifestModel]) -> None:
+	def enableOverrideIncompatibilityForAddon(
+			self,
+			listItemVM: AddonListItemVM[_AddonManifestModel],
+			askConfirmation: bool = True,
+			useRememberChoiceCheckbox: bool = False,
+	) -> tuple[bool, bool]:
 		from ... import mainFrame
-		if _shouldEnableWhenAddonTooOldDialog(mainFrame, listItemVM.model):
+		if askConfirmation:
+			res, shouldRememberChoice = _shouldEnableWhenAddonTooOldDialog(
+				mainFrame,
+				listItemVM.model,
+				
+				useRememberChoiceCheckbox=useRememberChoiceCheckbox,
+			)
+		else:
+			res = True
+			shouldRememberChoice = True
+		if res:
 			listItemVM.model.enableCompatibilityOverride()
 			self._handleEnableDisable(listItemVM, True)
+		return res, shouldRememberChoice
 
 	def enableAddon(self, listItemVM: AddonListItemVM) -> None:
 		self._handleEnableDisable(listItemVM, True)
 
+	def enableAddons(self, listItemVMs: Iterable[AddonListItemVM[_AddonStoreModel]]) -> None:
+		shouldEnableIncompatible = True
+		shouldRememberChoice = False
+		for aVM in listItemVMs:
+			if aVM.status in (
+				AvailableAddonStatus.INCOMPATIBLE_DISABLED,
+				AvailableAddonStatus.PENDING_INCOMPATIBLE_DISABLED,
+			) and not aVM.model.isCompatible and aVM.model.canOverrideCompatibility:

```

Comment:
```suggestion
			if aVM.canUseEnableOverrideIncompatibilityAction() and not aVM.model.isCompatible:
```

Reply:
the change made by you in response to this review comment dropped the usage of `and not aVM.model.isCompatible`, is this intentional?